R/utils.R

```diff
@@ -148,7 +148,7 @@
         stop("'MARGIN' must equal 1 or 2.", call. = FALSE)
```

Comment:
You could add parameter that specifies the name of the parameter.

Reply:
Check here name parameter as reference https://github.com/microbiome/mia/blob/c17fa9f70f556458247dda22112b14c9889efd3b/R/utils.R#L72